packages/react-native-worklets/src/shareables.ts

```diff
@@ -229,20 +233,24 @@ function makeShareableCloneRecursiveNative<T>(
 
 if (globalThis._WORKLETS_BUNDLE_MODE) {
   // TODO: Do it programatically.
-  makeShareableCloneRecursiveNative.__bundleData = {
-    imported: 'makeShareableCloneRecursive',
+  createSerializableRecursiveNative.__bundleData = {
+    imported: 'createSerializableRecursive',
     // @ts-expect-error resolveWeak is defined by Metro
     source: require.resolveWeak('./index'),
   };
 }
 
-export interface MakeShareableClone {
-  <T>(value: T, shouldPersistRemote?: boolean, depth?: number): ShareableRef<T>;
+export interface CreateSerializable {
+  <T>(
+    value: T,
+    shouldPersistRemote?: boolean,
+    depth?: number
+  ): SerializableRef<T>;
```

Comment:
Let's hide the two latter parameters so they wouldn't become a part of the public API.

We should probably retain old `MakeShareableClone` type to properly type cast for deprecated exports.

```suggestion
export interface CreateSerializable {
  <T>(
    value: T
  ): SerializableRef<T>;
```